inc/languages/english/admin/global.lang.php

```diff
@@ -52,6 +52,7 @@
 $l['error_invalid_ip'] = "Your IP address is not valid for this session.";
 $l['error_mybb_admin_lockedout'] = "This account has been locked out.";
 $l['error_mybb_admin_lockedout_message'] = "Your account is currently locked out after failing to login {1} times. You have been sent an email with instructions on how to unlock your account.";
+$l['error_mybb_not_admin_account'] = "Username is not an administrator.";
```

Comment:
This error message sounds strange and does not match the other messages. IMHO a more generic "You do not have the permission..." would be better.

Reply:
I think a generic error doesn't supply enough information. How about something like: "The account with which you attempted to log in does not have administrative privileges."?

Reply:
A login attempt of a non admin should always throw a 404. Why we reveal the route of admin login is correct?